tests/utils.go

```diff
@@ -243,6 +243,9 @@ type ObjectEventWatcher struct {
 	dontFailOnMissingEvent bool
 }
 
+const PromptExpression = "\\$ |\\# "
+const TermNLExpression = "\r\n"
```

Comment:
it named CRLF if you want , its righter than NL imo

Reply:
done